test/parallel/test-net-better-error-messages-port.js

```diff
@@ -3,12 +3,14 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT);
-
+const server = net.createServer();
+server.listen(0);
+const port = server.address().port;
+const c = net.createConnection(port);
 c.on('connect', common.mustNotCall());
-
-c.on('error', common.mustCall(function(e) {
+c.on('error', common.mustCall((e) => {
   assert.strictEqual(e.code, 'ECONNREFUSED');
-  assert.strictEqual(e.port, common.PORT);
+  assert.strictEqual(e.port, port);
   assert.strictEqual(e.address, '127.0.0.1');
 }));
+server.close();
```

Comment:
Since we've reached a moo point. Would it be alright if I followed @thefourtheye approach and change the code accordingly ?